.evergreen/scripts/resync-all-specs.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# Run spec syncing script and create PR
+
+# SETUP
+SRC_URL="https://github.com/mongodb/specifications.git"
+# needs to be set for resunc-specs.sh
+SPEC_SRC="$(realpath -s "../specifications")"
+SCRIPT="$(realpath -s "./.evergreen/resync-specs.sh")"
+
+# Clone the spec repo if the directory does not exist
+if [[ ! -d $SPEC_SRC ]]; then
+  git clone $SRC_URL $SPEC_SRC
+  if [[ $? -ne 0 ]]; then
+    echo "Error: Failed to clone repository."
+    exit 1
+  fi
+fi
+
+# Set environment variable to the cloned spec repo for resync-specs.sh
+export MDB_SPECS="$SPEC_SRC"
+
+# Check that resync-specs.sh exists and is executable
+if [[ ! -x $SCRIPT ]]; then
+  echo "Error: $SCRIPT not found or is not executable."
+  exit 1
+fi
+
+PR_DESC="spec_sync.txt"
+
+# run python script that actually does all the resyncing
+/opt/devtools/bin/python3.11 ./.evergreen/scripts/resync-all-specs.py "$PR_DESC"
```

Comment:
Same question about running locally

Reply:
Yeah. similar to the `resync-spec.sh` script, I'm using `if ! [ -n "${CI:-}" ]` to determine if its run locally vs on evergreen, is that okay? or is there a better way?

Reply:
That makes sense to me, thanks!